packages/image_picker/image_picker/README.md

```diff
@@ -41,6 +41,9 @@ It is no longer required to add `android:requestLegacyExternalStorage="true"` as
 **Note:** Images and videos picked using the camera are saved to your application's local cache, and should therefore be expected to only be around temporarily.
 If you require your picked image to be stored permanently, it is your responsibility to move it to a more permanent location.
 
+
+**Note:** Launching the image picker from an `Activity` with `launchMode: singleInstance` will always return `RESULT_CANCELED`. In this launch mode, new activities are created in a separate [Task][2]. As activities cannot communicate between tasks, the image picker activity cannot send back its eventual result to the calling activity. To work around this problem, consider using `launchMode: singleTask` instead.
```

Comment:
Now that we have 2 `Note`s, maybe it would be a good idea to add a section for `Critical Information` or something like that, to avoid collecting multiple "tag" endings on this section.

Reply:
Sounds good. Would that still fall under the Android heading? Example:

```
### Android

...

#### Critical information
Images and videos picked using the camera are saved to your application's local cache, and should therefore be expected to only be around temporarily.
If you require your picked image to be stored permanently, it is your responsibility to move it to a more permanent location.

Launching the image picker from an `Activity` with `launchMode: singleInstance` will always return `RESULT_CANCELED`.
In this launch mode, new activities are created in a separate [Task][2]. As activities cannot communicate between tasks, the image picker activity cannot send back its eventual result to the calling activity.
To work around this problem, consider using `launchMode: singleTask` instead.
```

Reply:
Yes, a subheading of `Android` as you've shown there would definitely be the right approach.

Reply:
I think @stuartmorgan has the right idea here.